File: BeagleUI/Source/UI/StyledWidget.swift
//
//  StyledView.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

struct StyledWidget: Widget {
    let border: Border?
    let color: String?
    let child: Widget?
}

struct Border {
    let color: String?
    let radius: Double?
    let size: Double?
}

File: beagle-ui/src/main/java/br/com/zup/beagleui/framework/widget/ui/StyledWidget.kt
package br.com.zup.beagleui.framework.widget.ui

import br.com.zup.beagleui.framework.widget.core.Widget

data class Border (
    val color: Long? = null,
    val radius: Double? = null,
    val size: Double? = null
)

data class StyledWidget (
    val border: Border? = null,
    val color: Long? = null,
    val child: Widget
) : Widget
